deps/chakrashim/include/v8.h

```diff
@@ -1765,15 +1772,15 @@ enum class ConstructorBehavior { kThrow, kAllow };
 class V8_EXPORT Function : public Object {
  public:
   static MaybeLocal<Function> New(Local<Context> context,
-                                  FunctionCallback callback,
-                                  Local<Value> data = Local<Value>(),
-                                  int length = 0,
-                                  ConstructorBehavior behavior = ConstructorBehavior::kAllow);
+    FunctionCallback callback,
```

Comment:
super nit:  Can you bring the first parameter down and indent them all two more spaces?